modules/analysis/src/endpoints.rs

```diff
@@ -34,12 +35,13 @@ pub fn configure(config: &mut utoipa_actix_web::service_config::ServiceConfig, d
 #[get("/v1/analysis/status")]
 pub async fn analysis_status(
     service: web::Data<AnalysisService>,
+    db: web::Data<Database>,
     user: UserInformation,
     authorizer: web::Data<Authorizer>,
     _: Require<ReadSbom>,
 ) -> actix_web::Result<impl Responder> {
     authorizer.require(&user, Permission::ReadSbom)?;
-    Ok(HttpResponse::Ok().json(service.status(()).await?))
+    Ok(HttpResponse::Ok().json(service.status(&**db).await?))
```

Comment:
I love concise code, but is this a tad too magic? I can't know at a glance what `&**db` is doing. Are we taking a reference of the result of calling first the `Deref` impl of `web::Data` and then that of `Database`? I'm not a rust noob, and I'm not sure. :smile: 

Is something like `service.status(db.get_ref().connection())` too wordy or less idiomatic?

Reply:
> Are we taking a reference of the result of calling first the `Deref` impl of `web::Data` and then that of `Database`?

Btw, close: The first `Deref` takes the `Arc` from `web::Data`, the second takes `T` (`Database`) from the `Arc`.

Reply:
Replaced this with `db.as_ref()`.